x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/detection_alerts/assignments/assignments.cy.ts

```diff
@@ -249,41 +206,39 @@ describe('Alert user assignment - ESS & Serverless', { tags: ['@ess', '@serverle
   });
 
   context('Updating assignees (bulk actions)', () => {
```

Comment:
Ditto about the "updating assignees" tests here: if both the original assignees and the updated assignees are the same, we're not really testing the "update" piece.